docs/concepts.rst

```diff
@@ -9,55 +9,71 @@ Let's start with some terms.
 Dependency
 ==================
 
-**Dependency** is what you need for some part of your code to work.
-If you *database gateway* needs *database connection* to execute SQL queries, then the connection is a dependency for a gateway. If your business logic class requires *database gateway* or some *api client* then *api client* and *database gateway* are dependencies for a business logic. Here the ``Client`` is a dependency, while ``Service`` is a dependant.
+**Dependency** is what you need for some parts of your code to work.
+If your *database gateway* needs *database connection* to execute SQL queries, then the connection is a dependency for a gateway.
+If your business logic class requires *database gateway* or some *api client* then *api client* and *database gateway* are dependencies for a business logic.
+Here, the ``Client`` is a dependency, while ``Service`` is a dependant.
 
 .. code-block:: python
 
     class Service:
         def __init__(self, client: Client):
             self.client = client
 
-So, dependency is just object required for some other object. Dependencies can depend on other objects, which are their dependencies.
+So, dependency is just an object required by another object. Dependencies can depend on other objects, which are their dependencies.
 
-To follow dependency injection rule, dependent object should receive their dependencies and not request themselves. The same classes can be instantiated with different values of their dependencies. At least in tests.
+To follow dependency injection rule, dependent objects should receive their dependencies and not request them themselves.
+The same classes can be instantiated with different values of their dependencies.
+At least in tests.
 
 
 Scope
 ===========
 
 **Scope** is a lifespan of a dependency.
-Some dependencies can live while application is running, others are created and destroyed on each request. In more rare cases you need more short-lived objects. You set a scope for your dependency when you configure how to create it.
+Some dependencies live for the entire application lifetime, while others are created and destroyed with each request.
+In more rare cases, you need more short-lived objects.
+You set a scope for each dependency when you configure how it is created.
 
-Standard scopes are (excluding skipped):
+Standard scopes are (with some skipped):
 
-    ``APP`` |rarr| ``REQUEST`` |rarr| ``ACTION`` |rarr| ``STEP``
+    ``APP`` |rarr| ``REQUEST`` |rarr| ``ACTION`` |rarr| ``STEP``.
```

Comment:
Why?

Reply:
Same in the README. Should we remove it in both places or only here?

Reply:
Formatting features are different. Let's remove trailing dot at least here

Reply:
Sure